### PR TITLE
Add `postLightingColor` property to materials

### PR DESCRIPTION
**Pull request by romainguy**
This property can be used to modify the color computed in the lighting
passes of the materials. That color is blended with the computed color
according to the postLightingBlending option (add, transparent or opaque).

### CODE REVIEW

File: tools/matc/src/matc/ParametersProcessor.cpp [hunk 1]
```diff
@@ -387,6 +391,30 @@ bool ParametersProcessor::processBlending(filamat::MaterialBuilder& builder,
...
+    }
+
+    filament::BlendingMode postLightingBlending =
+            stringToEnum(mStringToBlendingMode, jsonString->getString());
+    switch (postLightingBlending) {
+        case filament::BlendingMode::FADE:
+        case filament::BlendingMode::MASKED:
+            std::cerr << PARAM_KEY_BLENDING <<
+                    ": value is not a valid post-lighting BlendMode."
+                    " Only OPAQUE, TRANSPARENT and FADE are supported." << std::endl;
```
Comment (bejado) [inline tools/matc/src/matc/ParametersProcessor.cpp]: Shouldn't it be "OPAQUE, TRANSPARENT, and ADD"?

File: docs/Materials.md.html [hunk 2]
```diff
@@ -858,6 +879,30 @@
 }
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+### postLightingBlending
+
+Type
+:    `string`
+
+Value
+:     Any of `opaque`, `transparent`, `add`. Defaults to `opaque`.
```
Comment (bejado) [inline docs/Materials.md.html]: Would defaulting to "transparent" or "add" make more sense? To me, opaque seems like the least useful of the 3.

Comment (romainguy) [reply docs/Materials.md.html]: Yeah I've been debating that, the main reason I chose opaque as the default was to be consistent with the other blending option. It's also obvious what happens by default. Not sure which of transparent or add would be the better default.

File: tools/matc/src/matc/ParametersProcessor.cpp (hunk 1 above)
Comment (romainguy) [reply tools/matc/src/matc/ParametersProcessor.cpp]: Indeed

File: docs/Materials.md.html (hunk 2 above)
Comment (romainguy) [reply docs/Materials.md.html]: I'll switch the default to transparent.

Comment (bejado) [reply docs/Materials.md.html]: 👍 sounds good

**Pull request merged**